WordPress/UITestsFoundation/Screens/Login/Unified/PasswordScreen.swift

```diff
@@ -29,7 +29,7 @@ public class PasswordScreen: ScreenObject {
     @discardableResult
     public func proceedWithValidPassword() throws -> LoginEpilogueScreen {
         try tryProceed(password: "pw")
-
+        app.dismissSavePasswordPrompt()
```

Comment:
@jostnes  I'm surprised by this. `tryProceed(password:)` already calls it. 

Did it make a difference or is it an experiment?

Reply:
oh yeah, this has been removed, i overlooked this earlier, this was also the reason the test took a longer time to complete (as it was doing it twice, the second time completely unnecessary 😅)